packages/Liquid-Core.package/LiquidParticipantMenu.class/instance/sendAnswer.st

```diff
@@ -1,3 +1,7 @@
 buttons
 sendAnswer
-	self poll addAnswerSet: answerSet.
\ No newline at end of file
+	(LiquidParticipantMenu answeredPollIds ifAbsentAdd: self poll id)
+		ifTrue: [self poll addAnswerSet: answerSet]
+		ifFalse: [UserDialogBoxMorph inform: 'You have already answered this Poll.' title: 'Invalid operation!'].
+	self destroy.
+	^ ToolBuilder open: LiquidStartMenu new
```

Comment:
[dust] this smells like hard-coding, it may be interesting to have a `onSendAnswer` callback or something that decides what to do once the answer was provided. but it seems fine for now

Reply:
Since I just saw this in my notifications: Remember to use `UIManager default` instead of `UserDialogBoxMorph` to stick with how we do it in the rest of the code. Note however, that `UIManager default` doesn't support the `title:`-argument

Reply:
> [dust] this smells like hard-coding, it may be interesting to have a `onSendAnswer` callback or something that decides what to do once the answer was provided. but it seems fine for now

What do you mean exactly? Returning the startmenu for the test? I agree this isn't elegant, but I don't see where else we would call sendAnswer